build/common.sh

```diff
@@ -49,8 +49,7 @@ readonly KUBE_GCS_DELETE_EXISTING="${KUBE_GCS_DELETE_EXISTING:-n}"
 # Constants
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
 readonly KUBE_BUILD_GOLANG_VERSION=1.4.2
```

Comment:
OK, it's used by `kube::build::ensure_golang`, which is only called by `kube::build::build_image_cross`.
So when you're removing `kube::build::build_image_cross`, could you also remove `kube::build::ensure_golang` and `KUBE_BUILD_GOLANG_VERSION`?